### PR TITLE
[Storage] Tiny changes.

### PR DESCRIPTION
**Pull request by grao1991**
### Description
V -> v
256 -> 16

### CODE REVIEW

File: storage/aptosdb/Cargo.toml [hunk 1]
```diff
@@ -43,6 +43,7 @@ itertools = { workspace = true }
 lru = { workspace = true }
 move-core-types = { workspace = true }
 move-resource-viewer = { workspace = true }
+num_cpus = { workspace = true }
```
Comment (msmouse) [inline storage/aptosdb/Cargo.toml]: commit message needs fix

File: storage/aptosdb/src/state_store/mod.rs [hunk 2]
```diff
@@ -583,7 +584,7 @@ impl StateStore {
         first_version: Version,
         expected_usage: StateStorageUsage,
         ledger_batch: &SchemaBatch,
-        sharded_state_kv_batches: &[SchemaBatch; 256],
+        sharded_state_kv_batches: &[SchemaBatch; NUM_STATE_SHARDS],
```
Comment (msmouse) [inline storage/aptosdb/src/state_store/mod.rs:587]: `type SharedStateKvSchemaBatch = ... ` would be nice.

File: storage/aptosdb/Cargo.toml (hunk 1 above)
Comment (grao1991) [reply storage/aptosdb/Cargo.toml]: Done

File: storage/aptosdb/src/state_store/mod.rs (hunk 2 above)
Comment (grao1991) [reply storage/aptosdb/src/state_store/mod.rs:587]: Done

**Review by msmouse — APPROVED** [review]

**Pull request merged**